src/domains/my/detail/MyDrafts.jsx

```
@@ -1,28 +1,50 @@
+  const [isLoading, setIsLoading] = useState(false)
+  const [error, setError] = useState(null)
 
   useEffect(() => {
-    loadDrafts()
-  }, [loadDrafts])
+    const fetchDrafts = async () => {
+      setIsLoading(true)
+      setError(null)
+      try {
+        await loadFreeDrafts()
+      } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(no-unused-vars)